akka-docs/src/main/paradox/typed/reliable-delivery.md

```diff
@@ -28,13 +28,13 @@ To use reliable delivery, add the module to your project:
 Normal @ref:[message delivery reliability](../general/message-delivery-reliability.md) is at-most once delivery, which
 means that messages may be lost. That should be rare, but still possible.
 
-For interactions between some actors that is not acceptable and at-least once delivery or effectively once processing
+For interactions between some actors, that is not acceptable and at-least once delivery or effectively-once processing
 is needed. The tools for reliable delivery described here help with implementing that. It can't be achieved
-automatically under the hood without collaboration from the application because confirming when a message has been
+automatically under the hood without collaboration from the application, because confirming when a message has been
```

Comment:
Here it can be broken into two sentences perhaps, to avoid a run-on. For example
 ```suggestion
automatically under the hood without collaboration from the application. This is because confirming when a message has been```
```